src/main/resources/templates/start.html

```diff
@@ -1,77 +1,90 @@
- 
- <title>Where To Eat</title>
+	<meta charset="utf-8">
+  	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+ 	<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  	<link rel="stylesheet" type="text/css" th:href="@{/css/style.css}"/>
+	<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+ 	<title>Where To Eat</title>
 </head>
 <body>
-
-<h1>Where to Eat?!</h1><br/>
-<h2>Helping you solve the every day problem of "where to eat?"</h2>
-
-<h2>Click the Random button to pick a random restaurant near you.</h2>
-<div style="text-align:center;">
-	<button class="button">Random!</button> 
-</div>
-<br/><br/>
-<h2>Click the top 3 button to get top 3 random restaurants!</h2>
-
-<div style="text-align:center;">
-<button class="button">Top 3!</button>
-</div>
-<br/><br/>
-
-<h3>Feel like filtering your options? Customize your options, click the filter button then click Random!</h3>
-
-<select name="Food Type" class="type dropdown">
-  <option value="">Food Type</option>
-  <option value="1">Chinese</option>
-  <option value="2">Italian</option>
-  <option value="3">Asian (Includes all Asian countries)</option>
-  <option value="4">Indian</option>
-  <option value="5">Pizza</option>
-  <option value="6">Fast-Food</option>
-  <option value="7">Cafe</option>
-</select>
-<br/>
-
-<select name="Price Range" class="Price dropdown">
-  <option value="">Price Range</option>
-  <option value="1">$</option>
-  <option value="2">$$</option>
-  <option value="3">$$$</option>
-  <option value="4">$$$$</option>
-</select>
-<br/>
-
-<select class="Busy dropdown">
-  <option value="">Busy Range</option>
-  <option value="1">Not Busy</option>
-  <option value="2">Slightly Busy</option>
-  <option value="3">Busy</option>
-  <option value="4">Very Busy</option>
-</select>
-<br/>
-
-<select class="Ratings dropdown">
-  <option value="">Ratings Range</option>
-  <option value="1">1 Star +</option>
-  <option value="2">2 Star +</option>
-  <option value="3">3 Star +</option>
-  <option value="4">4 Star +</option>
-</select>
-<br/>
-
-<form action="/action_page.php">
-  Distance:<br>
-  <input type="text" name="distance" value="distance">
-  <br>
-</form> 
-
+ 	<!-- BANNER -->

```

Comment:
Updated HTML. Run the server and see the magic 😆 . Also sorry for the tab indentation. I tried to config it as 2 space but failed 😞